images/macos/provision/core/colima.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e -o pipefail
```

Comment:
Remove this file and use brew section in toolset file - https://github.com/actions/runner-images/blob/main/images/macos/toolsets/toolset-12.json

Reply:
I switched to toolset json